modules/fts/pages/fts-query-types.adoc

```diff
@@ -379,13 +379,17 @@ A phrase query is also demonstrated by means of the Java SDK, in xref:2.7@java-s
 == Geospatial Queries
 
 _Geospatial_ queries return documents that each specify a geographical location.
-Each query contains either:
+The location-data provided by a geospatial query can be any of the following:
 
-* A single _longitude_-_latitude_ coordinate pair; and a _distance_ value, in miles, which determines a radius measured from the location specified by the coordinate pair.
-Documents are returned if they specify (by means of a longitude-latitude coordinate pair) a location that lies within the radius.
+* A location, specified as a longitude-latitude coordinate pair; and a distance, in miles.
+The location determines the center of a circle whose radius-length is the specified distance.
+Documents are returned if they reference a location within the circle.
 * Two longitude-latitude coordinate pairs.
-These are respectively taken to indicate the upper left and lower right corners of a bounding box.
-Documents are returned if they specify a location that lies within the bounding box.
+These are respectively taken to indicate the top left and bottom right corners of a _rectangular_ bounding box.
+Documents are returned if they reference a location within the box.
+* An array of three of more longitude-latitude coordinate pairs.
+Each of the pairs is taken to indicate one corner of a _polygonal_ bounding box.
```

Comment:
same bounding box comment..

Reply:
Thanks, done as per comments above.

Reply:
feels like the "_polygonal_ bounding box" is a bit confusing to others.. :-) 
what about _polygonal_ bounding area?  Its used in one more place.

Reply:
_polygonal_ bounding box. => polygon here too?